docs/Options.md

```diff
@@ -53,7 +53,7 @@ The setup scripts also allow for optional overrides of the following inputs whos
 
 - `--access` _(`"public" | "restricted"`)_: Which [`npm publish --access`](https://docs.npmjs.com/cli/commands/npm-publish#access) to release npm packages with (by default, `"public"`)
 - `--author` _(`string`)_: Username on npm to publish packages under (by default, an existing npm author, or the currently logged in npm user, or `owner.toLowerCase()`)
-- `--bin` _(`string`)_: `package.json` bin value to include for npx-style running.
+- `--bin` _(`string`)_: `package.json` bin value to include for npx-style running. (eg `"./bin/index.js"`) - this does not create any of folders or files; just references to them in the `package.json` bin property
```

Comment:
🤔 This is a lot of text for a list item, and still doesn't really cover the nuance or recommended usage here. How about:

* Trimming this down a bit to more precisely just describe it as a `package.json` bin property string
* Writing an FAQ with more information & suggested way to do it

? That way we get the idea of _"export a `bin()`"_ suggested to users, without forcing them to do anything.

```suggestion
- `--bin` _(`string`)_: value to set in `package.json`'s `"bin"` property, per [FAQs > How can I use `bin`?]( ... )
```